### PR TITLE
hypervisor/host capabilities

### PR DESCRIPTION
**Pull request by shwstppr**
Note on hypervisor/host capabilities.
Addresses https://github.com/apache/cloudstack/issues/3015

### CODE REVIEW

**Comment by shwstppr** [comment]
@blueorangutan docbuild

**Comment by shwstppr** [comment]
@blueorangutan docbuild

File: source/adminguide/hosts.rst [hunk 1]
```diff
@@ -207,6 +207,22 @@ any system that is not up to date with patches.
...
+Following hypervisor-specific documentations can be referred for different maximums for a particular hypervisor host:
+
+- VMware: `VMware Configuration Maximum tool <https://configmax.vmware.com/guest?vmwareproduct=vSphere&release=vSphere%207.0&categories=1-0,2-0>`_.
+
+- Citrix Hypervisor/Xenserver/XCP-ng: `Configuration limits | Citrix Hypervisor 8.2 <https://docs.citrix.com/en-us/citrix-hypervisor/system-requirements/configuration-limits.html>`_.
+
+
+.. note::
+   Guest VM limit check is not done while deploying a VM on a KVM hypervisor host.
+
```
Comment (sureshanaparti) [inline source/adminguide/hosts.rst:223]: The number of guest VMs can be updated using _updateHypervisorCapabilities_ API. @shwstppr  Do you mentioned it, and about other APIs for hypervisor capabilities (_listHypervisorCapabilities_, _updateHypervisorCapabilities_).

Comment (shwstppr) [reply source/adminguide/hosts.rst:223]: @sureshanaparti as you can see in diff I've not mentioned APIs. I'll make the changes

**Comment by shwstppr** [comment]
@blueorangutan docbuild

**Pull request merged**